codesystems/CodeSystem-England-ORDOrganisationRole.xml

```diff
@@ -0,0 +1,31 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+  <id value="England-ORDOrganisationRole" />
+  <url value="https://ord.nhs.uk/CodeSystem/role" />
+  <version value="1.0.0" />
+  <name value="EnglandORDOrganisationRole" />
+  <title value="England ORD Organisation Role" />
+  <status value="draft" />
+  <date value="2023-11-16" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <contact>
+    <name value="Organisation Data Service" />
+    <telecom>
+      <system value="email" />
+      <value value="exeter.helpdesk@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="A set of codes that define the organisation roles that give added information for an organisation/site's function, for the Organisation Data Service (ODS).&#13;&#13;For more information and an up to date list see: [Organisation Data Service CodeSet](https://digital.nhs.uk/services/organisation-data-service/codesystems)." />
```

Comment:
There is comma towards the end of the sentence which probably should be removed. Also the bit after the comma doesn't quite make sense so may want to reword it a bit.

Reply:
Reworded based on input from ODS, but at under 24 hours before the sprint, a comma in a draft profile should not be hold this up.